cumulus/parachains/runtimes/bridge-hubs/bridge-hub-rococo/src/xcm_config.rs

```diff
@@ -402,19 +402,19 @@ pub struct XcmExportFeeToRelayerRewardAccounts<
 >(PhantomData<(AssetTransactor, DestNetwork, DestParaId, DestBridgedChainId, BridgeLaneId)>);
 
 impl<
-	AssetTransactor: TransactAsset,
-	DestNetwork: Get<NetworkId>,
-	DestParaId: Get<cumulus_primitives_core::ParaId>,
-	DestBridgedChainId: Get<ChainId>,
-	BridgeLaneId: Get<LaneId>,
-> HandleFee
-for XcmExportFeeToRelayerRewardAccounts<
-	AssetTransactor,
-	DestNetwork,
-	DestParaId,
-	DestBridgedChainId,
-	BridgeLaneId,
->
+		AssetTransactor: TransactAsset,
+		DestNetwork: Get<NetworkId>,
+		DestParaId: Get<cumulus_primitives_core::ParaId>,
+		DestBridgedChainId: Get<ChainId>,
+		BridgeLaneId: Get<LaneId>,
+	> HandleFee
+	for XcmExportFeeToRelayerRewardAccounts<
```

Comment:
Hmm, I'm not so sure about making these formatting changes, especially as its not our code. I want to reduce the amount of code that Parity needs to review.

Reply:
Actually it's a fix for https://github.com/paritytech/polkadot-sdk/pull/2522/files#r1430811135 which was introduced in `merge-to-upstream`.

Reply:
cool